### PR TITLE
chore: update Readme

### PR DESCRIPTION
**Pull request by roberto-butti**
I update the readme , adding:
- getting started
- fetching content
- enabling the Storyblok bridge
- load components
- rendering rich text
- rendering components into rich text
- enabling SSL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,8 +34,8 @@
 `@storyblok/svelte` helps you connect your Svelte project to Storyblok by:
 
 - Providing the `getStoryblokApi` function to interact with the Storyblok APIs, using the [storyblok-js-client](https://github.com/storyblok/storyblok-js-client)
-- Enabling real time editing through the [Storyblok Bridge](https://www.storyblok.com/docs/Guides/storyblok-latest-js?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-svelte)
-- Providing the `StoryblokComponent` which allows you to connect your components to the Storyblok Visual Editor
+- Enabling real-time editing through the [Storyblok Bridge](https://www.storyblok.com/docs/Guides/storyblok-latest-js?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-svelte)
+- Providing the `StoryblokComponent`, which allows you to connect your components to the Storyblok Visual Editor
```
Comment (josefineschaefer) [inline README.md:38]: We could link the Visual Editor Docs here: https://www.storyblok.com/docs/editor-guides/visual-editor

File: README.md [hunk 2]
```diff
@@ -48,7 +45,7 @@ Install `@storyblok/svelte`
 npm install @storyblok/svelte
 ```
 
-Please note that you have to use npm - unfortunately we are currently not supporting yarn or pnpm for this SDK.
+Please note that you have to use `npm` - unfortunately, we are currently not supporting yarn or `pnpm` for this SDK.
```
Comment (josefineschaefer) [inline README.md:48]: for consistency, let's put yarn in back ticks as well `yarn`

File: README.md [hunk 3]
```diff
@@ -74,13 +71,265 @@ storyblokInit({
 });
```
Comment (josefineschaefer) [inline README.md:71]: Is this only true for the US or also for China? If it's also true for China, let's be more generic to include all options here e.g. "if you are using a space located in a region other than the EU (default) use apiOptions.region to specify e.g. "us" or "cn"

File: README.md [hunk 4]
```diff
@@ -74,13 +71,265 @@ storyblokInit({
...
+
+```html
+<StoryblokComponent {blok} />
+```
+
+> The `blok` is the actual blok data coming from [Storblok's Content Delivery API](https://www.storyblok.com/docs/api/content-delivery/v2?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-svelte).
+
+### Features and API
+
+You can **choose the features to use** when you initialize the plugin. In that way, you can improve Web Performance by optimizing your page load and saving some bytes.
```
Comment (josefineschaefer) [inline README.md:149]: I think we can spell web performance with small letters

File: README.md [hunk 5]
```diff
@@ -74,13 +71,265 @@ storyblokInit({
...
+sbBridge.on(["input", "published", "change"], (event) => {
+  // ...
+});
+```
+
+For background information on the [Storyblok JS Bridge](https://www.storyblok.com/docs/Guides/storyblok-latest-js?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-svelte), please check out the documentation.
+
+### Rendering Rich Text
+
+You can easily render rich text by using the `renderRichText`function that comes with `@storyblok/svelte`and Sveltes `{@html htmlstring}`directive.
```
Comment (josefineschaefer) [inline README.md:191]: There are spaces missing after "renderRichText", "@storyblok/svelte" and "htmlstring"

File: README.md [hunk 6]
```diff
@@ -92,9 +341,9 @@ The parts are:
...
-This project uses [semantic-release](https://semantic-release.gitbook.io/semantic-release/) for generate new versions by using commit messages and we use the [Angular Convention](https://docs.google.com/document/d/1QrDFcIiPjSLDn3EL15IJygNPiHORgU1_OOAqWjiDU5Y/edit#) to naming the commits. Check [this question](https://semantic-release.gitbook.io/semantic-release/support/faq#how-can-i-change-the-type-of-commits-that-trigger-a-release) about it in semantic-release FAQ.
+This project uses [semantic-release](https://semantic-release.gitbook.io/semantic-release/) for generate new versions by using commit messages and we use the [Angular [Convention](https://docs.google.com/document/d/1QrDFcIiPjSLDn3EL15IJygNPiHORgU1_OOAqWjiDU5Y/edit#) to name the commits. Check [this question](https://semantic-release.gitbook.io/semantic-release/support/faq#how-can-i-change-the-type-of-commits-that-trigger-a-release) about it in semantic-release FAQ.
```
Comment (josefineschaefer) [inline README.md:349]: before Angular, there is a rouge [

File: README.md [hunk 7]
```diff
@@ -74,13 +71,265 @@ storyblokInit({
 });
 ```
 
-Add all your components to the components object in the storyblokInit function. You can load all of them at the same time by adding them to the list.
+The best place to initialize the Stroyblok library is in the `main.ts.` or `main.js` file if you are using `svelte` or in the `load()` function in the `src/routes/+layout.js` file if you are using `SvelteKit`.
+
+List all your components to the components object in the storyblokInit function. You can load all of them by adding them to the list.
+
+## Getting started
```
Comment (josefineschaefer) [inline README.md:78]: Should we maybe mention the vite version somewhere?

**Review by josefineschaefer — APPROVED** [review]
I added a few suggestions :)

File: README.md (hunk 3 above)
Comment (roberto-butti) [reply README.md:71]: i have to investigate more about the China configuration. So for now, let me add only the US option. Then i will open a new PR when EU vs US vs CN will be more clear to me

**Pull request merged**